hack/verify-vendor.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+WMCO_ROOT=$(pwd)
+
+# create a temporary location to put our new vendor tree
+mkdir -p "${WMCO_ROOT}/_tmp"
+_tmpdir="$(mktemp -d "${WMCO_ROOT}/_tmp/wmco-vendor.XXXXXX")"
+
+# Copy the contents of the WMCO directory into the temporary location
+_wmcotmp="${_tmpdir}/WMCO"
+mkdir -p "${_wmcotmp}"
+tar --exclude=.git --exclude="./_*" -c . | (cd "${_wmcotmp}" && tar xf -)
+
+export GO111MODULE=on
+
+pushd "${_wmcotmp}" > /dev/null 2>&1
+  # Destroy deps in the copy of the WMCO tree
+  rm -rf ./vendor 
+
+  # Recreate the vendor tree using the clean set we just downloaded
+  go mod vendor
```

Comment:
This sort of indentation around bash builtins like `pushd` and `popd` is not required and moreover is misleading. Please remove.

Reply:
Ok.